packages/typespec-vs/src/Microsoft.TypeSpec.VS.csproj

```diff
@@ -41,17 +41,23 @@
   </ItemGroup>
   <ItemGroup>
     <!-- Use 17.0.x or latest 16.x if no 17.0.x for compatible API-->
-    <PackageReference Include="Microsoft.VisualStudio.SDK" Version="17.0.31902.203"
-      ExcludeAssets="Runtime" />
+    <PackageReference Include="Microsoft.VisualStudio.SDK" Version="17.10.40171" ExcludeAssets="Runtime" />
+    <PackageReference Include="Microsoft.VisualStudio.SDK.Analyzers" Version="17.7.22">
```

Comment:
Added it because latest Microsoft.VSSDK.BuildTools would report warning as error that it can't find the expected version of Microsoft.VisualStudio.SDK.Analyzers unless I install the latest version of Microsoft.VisualStudio.SDK.Analyzers explicitly.... Anyway, I tried to update Microsoft.VSSDK.BuildTools to the latest but one version which is good enoug for us and doesn't have this issue. so remove this explicit package dependency.